lib/pkg.ml

```diff
@@ -561,20 +576,21 @@ type f =
   dir:Fpath.t ->
   args:Cmd.t ->
   out:(OS.Cmd.run_out -> (string * OS.Cmd.run_status, Sos.error) result) ->
+  ?err:Bos.OS.Cmd.run_err ->
```

Comment:
I wasn't sure here if to ignore the dune error output both in `check` and when building/testing in `distrib` or just in the first. Actually I think in both would be better, since the dune message pollutes the dune-release message a lot and `dune build`/`dune runtest` can be run separately if any of them fails. But I've opted for only in `check` to not change the output of `distrib` without necessity. What do you think?

Reply:
Ok, done.